Gemfile

```diff
@@ -3,6 +3,7 @@ source('https://rubygems.org')
 gemspec
 gem 'codecov', :require => false, :group => :test
 gem "danger", "~> 8.0"
+gem "bigdecimal", "~> 1.4"
```

Comment:
Constraining `bigdecimal` in the `Gemfile` will work for the repo, but downstream dependencies won't inherit this constraint, as can be seen in [the sister PR, which uses `bigdecimal 2.0.0`](https://github.com/Automattic/simplenote-macos/pull/657/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R48).

This can be problematic, as per my comment in the sister PR.

I pushed [a commit on another branch that moves this to the `.gemspec`](https://github.com/wordpress-mobile/release-toolkit/commit/f1605543b9d92bfe2a02ef4151a1ca77cb5414f4), feel free to cherry pick it.

If `simplenote-macos` uses that commit, the plugin gets loaded successfully.

Reply:
Good point @mokagio!
For some reasons, on my local environment the error went back to be a warning with my configuration, but this your solution seems better anyway. 
I've pushed it and updated the sister PR on Simplenote-MacOS